src/coreclr/tools/Common/Compiler/Win32Resources/ResourceData.Win32Structs.cs

```diff
@@ -84,7 +84,13 @@ public IMAGE_RESOURCE_DATA_ENTRY(ref BlobReader blobReader)
 
             public static void Write(ref ObjectDataBuilder dataBuilder, ISymbolNode node, int offsetFromSymbol, int sizeOfData)
             {
-                dataBuilder.EmitReloc(node, RelocType.IMAGE_REL_BASED_ADDR32NB, offsetFromSymbol);
+                dataBuilder.EmitReloc(node,
+#if READYTORUN
+                    RelocType.IMAGE_REL_BASED_ADDR32NB,
```

Comment:
It looks like we have an arbitrary difference in how these RelocTypes are used between crossgen vs. ilc. Which one is right?

(just asking, not something to fix in this PR)

Reply:
The one crossgen2 is using came from a projectn integration and never meant anything in nativeaot. But it looked like absolute has some meaning to crossgen2 so it's all confusing.